packages/themes/_btns.scss

```diff
@@ -272,3 +272,9 @@ button {
 .btn-hide {
   display: none !important;
 }
+
```

Comment:
@lebaphi  - could you explain what problem this next block overcomes, please?

Reply:
Hi @IanMayo, I just adjusted the button's color when it's disabled for ease of viewing, without this block, this button looks like:

<img width="395" alt="image" src="https://user-images.githubusercontent.com/26705621/207749884-285c41e2-1739-4239-bf6f-69d3da2f932a.png">

Reply:
@lebaphi - is there a reason why we don't make it hidden when disabled?
![image](https://user-images.githubusercontent.com/1108513/207865889-07558b02-40b4-4750-acb7-87049bc130bd.png)

Reply:
ah, i think disable should just update the state of the button instead of hiding it, it makes more sense. But if you want to hide this button, it's ok.

Reply:
Thanks for trying to "do the right thing".   In this instance, the `Delete` button isn't shown when the form is disabled, so I think it's logical to also not show the `Copy` button:
![screencast](https://i.gyazo.com/42c9dd41ddbe63510f86295a063ecde1.gif)

So, ok to remove?

Reply:
> So, ok to remove?

sure @IanMayo